skbio/alignment/_tabular_msa.py

```diff
@@ -22,6 +22,8 @@
 from skbio.util._decorator import experimental, classonlymethod, overrides
 from skbio.util._misc import resolve_key
 
+from skbio.alignment._tabular_msa_repr import _TabularMSAReprBuilder
```

Comment:
recommend renaming `skbio.alignment._tabular_msa_repr` to `skbio.alignment._repr` to match `skbio.sequence._repr`